.dockerignore

```diff
@@ -0,0 +1,7 @@
+*.db
+*.md
+.dockerfile
+.editorconfig
+.gitignore
+Dockerfile
+example/
```

Comment:
newline at EOF please

Reply:
This should be addressed

Reply:
For some reason, this isn't being reflected on the Github editor / viewer. But I've definitely added it.

Reply:
Looks good now, weirdly.